lms/templates/problem.html

```diff
@@ -5,38 +5,106 @@
 %>
 
 <%namespace name='static' file='static_content.html'/>
-<h3 class="hd hd-2 problem-header">
+<h3 class="hd hd-2 problem-header" id="${ id }-problem-title" aria-describedby="${ id }-problem-progress" tabindex="-1">
```

Comment:
@muzaffaryousaf I don't think that any of us were aware of short_id. What is the difference between the 2? Are you asking for us to change all id usages to short_id?

FYI @staubina